Basic/DyanmicPrefabs/Assets/Scripts/03_Server Authoritative Try Spawn Synchronously/SynchronousDynamicPrefabSpawn.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
+using Random = UnityEngine.Random;
+
+namespace Game.SynchronousDynamicPrefabSpawn
+{
+    /// <summary>
+    /// A dynamic prefab loading use case where the server instructs all clients to load a single network prefab, and
+    /// will only invoke a spawn once all clients have successfully completed their respective loads of said prefab. The
+    /// server will initially send a ClientRpc to all clients, begin loading the prefab on the server, will await
+    /// acknowledgement of a load via ServerRpcs from each client, and will only spawn the prefab over the network once
```

Comment:
nice level of details here :)
Could be great to add a line or two on the why? Or are we covering the "why" somewhere else?